test/extended/deployments/deployments.go

```diff
@@ -270,10 +270,26 @@ var _ = g.Describe("deploymentconfigs", func() {
 
 			g.By("deploying a few more times")
 			for i := 0; i < 3; i++ {
-				out, err = oc.Run("deploy").Args("--latest", "--follow", "deployment-test").Output()
-				o.Expect(err).NotTo(o.HaveOccurred())
+				rolloutCompleteWithLogs := make(chan struct{})
+				out := ""
+				go func(rolloutNumber int) {
+					var err error
+					defer func() {
+						close(rolloutCompleteWithLogs)
+						g.GinkgoRecover()
+					}()
+					out, err = waitForDeployerToComplete(oc, fmt.Sprintf("deployment-test-%d", rolloutNumber), deploymentRunTimeout)
```

Comment:
@kargakis this will make sure we wait up to 5 minutes for the deployer pod to be created and running instead of relying on the logs timeout (which is 1 minute).